### PR TITLE
server: add metrics for duration of each session execute stage

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -445,30 +445,37 @@ func (s *session) Execute(sql string) ([]ast.RecordSet, error) {
...
 		return nil, errors.Trace(err)
 	}
+	startTS := time.Now()
 	charset, collation := getCtxCharsetInfo(s)
 	rawStmts, err := s.ParseSQL(sql, charset, collation)
 	if err != nil {
 		log.Warnf("compiling %s, error: %v", sql, err)
 		return nil, errors.Trace(err)
 	}
+	sessionExecuteDuration.WithLabelValues(stageParse).Observe(float64(time.Since(startTS).Nanoseconds() / 1e6))
```
Comment (shenli) [inline session.go]: Use Seconds()

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
After some test, I find parse SQL takes 0.085ms on average and 95% within 0.15ms, 
query optimize takes 0.069ms on average and 95% within 0.12ms.

so, I change the Histogram buckets to more reasonable values...

PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**